ext/mbstring/mbstring.c

```diff
@@ -2941,6 +2941,144 @@ PHP_FUNCTION(mb_strtolower)
 	RETURN_STR(mbstring_convert_case(PHP_UNICODE_CASE_LOWER, ZSTR_VAL(str), ZSTR_LEN(str), enc));
 }
 
+typedef enum {
+	MB_LTRIM = 1,
+	MB_RTRIM = 2,
+	MB_BOTH_TRIM = 3
+} MB_TRIM_MODE;
+
+static bool is_trim_wchar(uint32_t w, HashTable *ht)
```

Comment:
```suggestion
static zend_always_inline bool is_trim_wchar(uint32_t w, const HashTable *ht)
```

Nit: Just in case the compiler is dumb; also const makes sure we don't accidentally modify it and shows the intent better to the callers.